certification_organism/data/product_product.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="__export__.product_product_3_ec1455d9" model="product.product" context="{'studio': True}">
+        <field name="product_tmpl_id" ref="__export__.product_template_3_cccbc90f"/>
+        <field name="standard_price">150.0</field>
+    </record>
+    <record id="industry_fsm_sale.field_service_product" model="product.product" context="{'studio': True}">
+        <field name="product_tmpl_id" ref="industry_fsm_sale.field_service_product_product_template"/>
+    </record>
+    <record id="sale_timesheet.time_product" model="product.product" context="{'studio': True}">
```

Comment:
you can't define records with xml_id from other module, remove the prefix sale_timesheet